tools/clang/lib/Format/UnwrappedLineParser.cpp

```diff
@@ -1525,7 +1525,8 @@ void UnwrappedLineParser::parseEnum() {
     nextToken();
 
   // Eat up enum class ...
-  if (FormatTok->Tok.is(tok::kw_class) || FormatTok->Tok.is(tok::kw_struct))
+  if (FormatTok->Tok.is(tok::kw_class) || FormatTok->Tok.is(tok::kw_struct) ||
+      FormatTok->Tok.is(tok::kw_union))
```

Comment:
I assume we don't support enum unions, so we probably don't need this change either.

Reply:
Done